compiler/luci-interpreter/src/kernels/Mean.test.cpp

```diff
@@ -27,6 +27,9 @@ namespace
 
 using namespace testing;
 
+// for quantized Add, the error shouldn't exceed step
+float GetTolerance(int min, int max) { return (max - min) / 255.0; }
```

Comment:
This is not just for quantized Add (see the comment).

Plus, min and max seem to be float, not int.

If GetTolerance is used in all quantized tests, it had better be placed in "kernels/TestUtils.h". Then, the function can have a more general form, e.g., `float getTolerance(float min, float max, int quantize_steps)`. `quantize_steps` will be 255 for uint8 quantization.

Reply:
Oh I will fix it!